docs/contributing/database-migrations/index.md

```diff
@@ -43,4 +49,23 @@ When writing a migration script there are a couple of best practices we follow.
 [repository]:
   https://docs.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design
 [dapper]: https://github.com/DapperLib/Dapper
-[code-style-sql]: ../../../../contributing/code-style/sql.md
+[code-style-sql]: ../code-style/sql.md
+
+## EF Migrations
```

Comment:
EF migrations are documented in https://contributing.bitwarden.com/getting-started/server/database/ef/. We should figure out a good way to organize these.